### ISSUE TITLE
Running this two times (to create two individual WordPress Dockers on a host machine) will lead to a conflicting database (due to the static volume and DB name)?

### ISSUE THREAD

**Issue by ColorWP** [eugeneware/docker-wordpress-nginx#39]
I noticed that you are mounting the mysql volume and always creating a DB named "wordpress" when this DockerFile is runned. What if I decide to run multiple WordPress installations with separate Dockers on a same host?

They will confluct due to the same DB name in my opinion - "wordpress".

**Comment by levino** [comment]
No. You should mount different host volumes into the containers. One uniquely for one container instance. Or just not even mount anything into the container. Having multiple mysql daemons work on the same files will most likely destroy your dbs.
